test/integration/confirmations/signatures/permit-seaport.test.tsx

```diff
@@ -112,6 +114,8 @@ describe('Permit Seaport Tests', () => {
   it('renders offer and consideration details', async () => {
     await renderSeaportSignature();
 
+    fireEvent.click(screen.getByTestId('sectionCollapseButton'));
+
     const offers = await screen.findByTestId('confirmation_data-offer-index-2');
```

Comment:
message section in seaport requests is not collapsed as we show simulation details.